xbmc/listproviders/DirectoryProvider.cpp

```diff
@@ -382,7 +382,7 @@ bool CDirectoryProvider::OnClick(const CGUIListItemPtr &item)
 {
   CFileItem fileItem(*std::static_pointer_cast<CFileItem>(item));
 
-  if (fileItem.HasVideoInfoTag()
+  if (fileItem.HasVideoInfoTag() && !fileItem.m_bIsFolder
```

Comment:
How can an item be a video (have a video info tag) and a folder at the same time?

Reply:
Well, TV Shows items has "Video Info Tag" and it is also a folder. 
Please, have a look at the two files attached, I believe it will be easier to understand the issue.

![Normal tv show path](https://user-images.githubusercontent.com/9671193/107495265-f6f60a80-6be3-11eb-86b3-a08d029f8595.png)
![Widget path](https://user-images.githubusercontent.com/9671193/107495279-fd848200-6be3-11eb-8e7c-3ba6197c590e.png)